R/control.R

```diff
@@ -70,6 +70,41 @@ print.control_grid <- function(x, ...) {
 #' @export
 control_resamples <- control_grid
 
+#' Control aspects of the last fit process
+#'
+#' @inheritParams control_grid
+#'
+#' @details
+#'
+#' [control_last_fit()] is a wrapper around [control_resamples()] and is meant
+#'   to be used with [last_fit()].
+#'
+#' @export
+control_last_fit <- function(
+    verbose = FALSE,
+    pkgs = NULL,
```

Comment:
Can you say more about why you included `pkgs` as an argument? Is this used for anything other than parallel processing? The other args that control parallel processing are not included, for good reason.

Reply:
No, I think you are right. The `pkgs` argument can probably be removed. Want me to push this change?

Reply:
Fixed with [cb41113](https://github.com/tidymodels/tune/pull/481/commits/cb41113a27ddccfa60890bc61a47e6b437a822b6)